packages/dso-toolkit/src/utilities.scss

```diff
@@ -50,9 +50,11 @@
   border: 0;
 }
 
-@mixin focus-outline() {
+@mixin focus-outline($offset: units.$focus-outline-offset) {
   border-radius: units.$focus-border-radius;
-  outline-offset: units.$focus-outline-offset;
+  @if $offset != 0 {
+    outline-offset: $offset;
+  }
   outline: units.$focus-outline-width solid colors.$focus-outline-color;
```

Comment:
Wil je dit nog houden?

Reply:
dat vroeg ik mij ook af, maar het antwoord is ja; in verband met de beruchte deprecation warnings is het beter om zaken binnen de mixin te overschrijven dan daarbuiten. Ook is het beter om de offset-property helemaal niet uit te schrijven als deze 0 (default) is.

Reply:
Ik vind dit geen YAGNI ;)

Reply:
Ok, tot op zekere hoogte ben ik het met je eens.

Maar.... In https://github.com/dso-toolkit/dso-toolkit/issues/2757#issuecomment-2291125345 sturen we juist aan op één focus styling.

We willen onszelf daarmee ook niet eens de ruimte geven om hier andere waardes te plaatsen. Het is altijd `units.$focus-outline-offset` (3px nu). Mocht dat ooit 0 worden, dan moet op dat moment de property gewoon in z'n geheel worden verwijderd.

Reply:
`focus-outline` is een hele subjectieve property, die erg afhankelijk is van de situatie ter plekke. Als je een mogelijkheid hebt om die meteen goed in te stellen zonder achteraf te moeten overschrijven, is dat toch wel heel gewenst. Zeker nu met die `& {}` ellende.

Ik zeg: laat lekker zo staan, als je geen args meestuurt gaat het gewoon als voorheen.